README.md

```diff
@@ -7,8 +7,12 @@
 ### This extension may conflict with dev version of Secure Shell extension, you may need to remove dev version of Secure Shell (if installed)
 
 - Download [archive](https://github.com/supechicken/ChromeOS-AutoStart/archive/main.zip) of this repository and unzip it
-- Go to chrome://extensions/ and enable Developer Mode
-- Click `Load unpacked extension` and select the `ChromeOS-AutoStart-main` unzipped folder
-- A new window will appear and type the command you want to run it at startup
-- Optional: Click the `Test` button to test it out after setting a command
-- Reboot and enjoy :)
+- Unpack the zip file by:
+  - Double click the zip file in the file manager. The zip file will show contents in what looks like a flash drive
+  - Drag the folder within the zip file into the Downloads folder
+  - Optionally delete the zip file
```

Comment:
Good idea :) I remember that a key file is generated alongside the crx file and we can remove it also

Reply:
Yes, feel free to add that.